src/utils.js

```diff
@@ -34,6 +34,8 @@ function getCardColorByType(cardType = "other") {
     case cards.amex:
       color = "rgb(0, 112, 209)";
       break;
+    case cards.maestro:
+      color = "rgb(196, 81, 225)";
```

Comment:
Please add `break` statement here ya @Beat-tech

Reply:
On it!